src/DispatchListener.php

```diff
@@ -216,7 +216,7 @@ protected function marshalBadControllerEvent(
         $controllerName,
         MvcEvent $event,
         Application $application,
-        \Exception $exception
+        \Throwable $exception
```

Comment:
@webimpress It works fine on PHP 5.6; the class does not need to exist for a typehint check.